modules/docs/src/Volo.Docs.Admin.Application/Volo/Docs/Admin/Documents/DocumentAdminAppService.cs

```diff
@@ -27,23 +27,23 @@ public class DocumentAdminAppService : ApplicationService, IDocumentAdminAppServ
         private readonly IDistributedCache<DocumentUpdateInfo> _documentUpdateCache;
         private readonly IDistributedCache<List<VersionInfo>> _versionCache;
         private readonly IDistributedCache<LanguageConfig> _languageCache;
-        private readonly IDocumentFullSearch _documentFullSearch;
+        private readonly IDocumentFullSearch _elasticSearchService;
```

Comment:
I think `_documentFullSearch` is a better field name since `_elasticSearchService` says something about the implementation and breaks encapsulation.